flink-contrib/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -834,7 +836,7 @@ private void restoreKVStateData() throws IOException, RocksDBException {
 	}
 
 	@Override
-	protected <N, T> InternalValueState<N, T> createValueState(
+	public <N, T> InternalValueState<N, T> createValueState(
```

Comment:
It is mainly due to the unit tests in `KvStateRequestSerializerTest` which need the accessors to `InternalKvState`.  A better choice to use `getPartitionState()` to obtain a user-facing state and convert it to an internal state. What do you think?